pkg/kn/commands/plugin/plugin_list.go

```diff
@@ -80,7 +80,9 @@ func listPlugins(cmd *cobra.Command, flags pluginListFlags) error {
 
 	if len(pluginsFound) == 0 {
 		if flags.verbose {
-			fmt.Fprintf(out, "No plugins found in path %s\n", pluginPath)
+			fmt.Fprintf(out, "No plugins found in path %s.\n", pluginPath)
```

Comment:
How about we remove the "in path..." part here since it makes for a really long and hard to read message, and if we still want to show the PATH (which I think is silly) then do it above at line 78 like all of the other settings we show.

Reply:
Actually this would show the whole path which has been used(including $PATH if enabled). this is not exactly the same as in 78 as this doesn't show the $PATH. And as this is written only on `--verbose` I think it could be even redundant.

As I guess you not a typical `--verbose` user ;-P, this shouldn't be so critical ?

Reply:
yep - I tend to be a minimalist by default, and as you've said, that's what --verbose is for.
So w/o any kind of --verbose flag I want to see the bare minimum info, just the facts. And showing me the PATH just clutters up the screen. It's only useful when things go wrong (which most of the time they won't), but when they do, that's when they can add --verbose and see all of the gory details of where we looked.